### PR TITLE
Add a web docs page explaining the API changes and how to update.

### PR DESCRIPTION
**Pull request by jatkinson1000**
This PR adds a web docs page explaining the API changes and how to update following the breaking changes of #136 and #143 

This can then be sent to known users to help them update.

### CODE REVIEW

**Comment by dorchard** [comment]
Alongside this perhaps we should bump the package number here: https://github.com/Cambridge-ICCS/FTorch/blob/ac4c14f307e79a147e377b8311ea40af4bd7ec34/src/CMakeLists.txt#L4
And perhaps a change to switch to [semantic versioning](https://semver.org/)?

**Comment by jatkinson1000** [comment]
We could, but I am not sure it carries much meaning since we haven't updated it so far...
My plans were to make a JOSS release v1.0.0 and then be more rigorous from that point.

**Review by TomMelt — APPROVED** [review]
@jatkinson1000 this is amazing. Thanks for doing this. It's really clear what the changes are and how to upgrade to the new API. Super :rocket:

**Comment by jatkinson1000** [comment]
Thanks @TomMelt I will wait for Joe to have a final look over #143 and then merge this after that goes in.

I'm collating a list of users to email so please add any users you are aware of that I need to email here @dorchard @jwallwork23:

- Laura Mansfield and Stanford
- Will Chapman and NCAR
- H. Heuer and ICON
- vbalmer
- WaveWatch 3

File: pages/updates.md [hunk 1]
```diff
@@ -0,0 +1,106 @@
...
+will be avoided. We hope that this is the last time we have such a shift.
+
+The changes allow us to implement two new features:
+
+#. Multiple output tensors
+   Previously you could pass an array of several input tensors to a torch model, but
+   only recieve a single output tensor back. Now you can use models that return several
+   output tensors by passing an array of output tensors instead.
+#. Preparation for autograd functionality
+   We hope to make it easier to access the autograd features of pytorch from within Fortran.
```
Comment (jwallwork23) [inline pages/updates.md:24]: PyTorch

**Review by jwallwork23 — APPROVED** [review]
Very clear, nice! One optional minor comment.

File: pages/updates.md (hunk 1 above)
Comment (jatkinson1000) [reply pages/updates.md]: Should be fixed in https://github.com/Cambridge-ICCS/FTorch/pull/144/commits/945e24a35c84192f25c0c8d1793d67dc08993ac3 Thanks!

**Comment by jatkinson1000** [comment]
Minor Aesthetic changes to the webpage and added a note to the README to highlight until v1.0.0.
Issue #146 opened to remind us to revert these changes after a suitable period.

**Pull request merged**